<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_addroom</class>
 <widget class="QDialog" name="Dialog_addroom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog_addroom
{
background-color:white;
}</string>
  </property>
  <widget class="QWidget" name="widget_top" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_top
{
border: 1px solid #000000;
background-color: rgb(157, 196, 229);
background-color: rgb(157, 196, 229);
}</string>
   </property>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>5</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/image/closeWindow.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>481</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget
{
border: 1px solid #000000;
}</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>169</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { 
border: none; 
background-color : rgb(105, 105, 105);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>141</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>141</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { 
border: none; 
color : rgb(105, 105, 105);
}</string>
       </property>
       <property name="text">
        <string>定义房间</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color:#ffffff; /*文字颜色*/
    background-color: #00ACB9;/*背景色*/
    border-radius:0px; /*边框倒角*/
	border: 1px solid #000000;
}


QPushButton:hover 
{
background-color: rgb(0,142,153);
color: white;
}</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>401</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_2
{
border: 1px solid #DCDCDC;
}

QLabel
{
font-family: '黑体';
font-size: 16px;
}

QLineEdit
{
font-family: '黑体';
}

QRadioButton
{
font-family: '黑体';
}

QPushButton
{
font-family: '黑体';
}</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_roomid">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>房间类型：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_limit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>260</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>噪声限值dB(A)：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_roomcalclass">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>310</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>主风管数量：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>房间名称：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_roomname">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_jiaban">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>房间编号：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pipe">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>房间计算类型：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>甲板：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_roomid</tabstop>
  <tabstop>lineEdit_roomname</tabstop>
  <tabstop>lineEdit_jiaban</tabstop>
  <tabstop>lineEdit_pipe</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>lineEdit_roomcalclass</tabstop>
  <tabstop>close</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>lineEdit_limit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
